odoo/addons/base/security/base_groups.xml

```diff
@@ -7,8 +7,14 @@
             Note that the field 'category_id' is set later in
             base/data/ir_module_category_data.xml
         -->
+        <record model="res.groups" id="group_partner_manager">
+            <field name="name">Contact Creation</field>
+            <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
+        </record>
+
         <record model="res.groups" id="group_erp_manager">
             <field name="name">Access Rights</field>
+            <field name="implied_ids" eval="[(4, ref('base.group_partner_manager'))]"/>
```

Comment:
On second thought, I'm not sure it's really needed to fix this. The default user has `group_partner_manager` anyway and the change may be a bit touchy.

Reply:
This is my feeling too.